build-system-tests/package.json

```diff
@@ -21,7 +21,7 @@
     "test:vue:nuxt": "./scripts/setup-mega-app.sh -F vue --build-tool nuxt",
     "setup:react-native": "./scripts/setup-mega-app.sh -F react-native",
     "setup:react-native:cli": "./scripts/setup-mega-app.sh -F react-native -B cli",
-    "test:react-native:expo": "./scripts/setup-mega-app.sh -F react-native -B expo",
+    "setup:react-native:expo": "./scripts/setup-mega-app.sh -F react-native -B expo",
```

Comment:
Info: if this is running the setup command, do you why was it labeled as "test"?

Reply:
I renamed to test in https://github.com/aws-amplify/amplify-ui/pull/5878 but doing web and RN separately so trying to revert this for now. `setup-mega-app` runs the whole build test suite so it's `test` is more accurate

Reply:
Ah, ok. So are we going to rename it to `test` again in the future?

Reply:
Yes